docs/ide-usage.md

```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
+
+       ![setupguide-7.png](images/setupguide-7.png)
```

Comment:
Apologies for that oversight on my part 😅 here's the required screenshot:

![screen shot 2017-04-19 at 12 33 51 am](https://cloud.githubusercontent.com/assets/7261051/25141978/eefd7ff8-2497-11e7-8606-5138f97072f8.png)